thuyetnv/thuyetnv/controller/controller_login.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$username = $_POST["username"];
+$password = md5($_POST["password"]);
+$check = fetch_one("select username, pass from user where username='$username'");
```

Comment:
Nên kiểm tra đồng thời username và password trong cùng 1 query để không phải viết thêm code kiểm tra ở PHP. `SELECT * FROM user WHERE username = '$username' AND password = '$password'`